ethportal-peertest/src/scenarios/basic.rs

```diff
@@ -114,13 +111,8 @@ pub async fn test_history_store(target: &Client) {
 
 pub async fn test_history_routing_table_info(target: &Client) {
     info!("Testing portal_historyRoutingTableInfo");
-    let result = HistoryNetworkApiClient::routing_table_info(target)
-        .await
-        .unwrap();
-    assert!(result.get("buckets").unwrap().is_object());
-    assert!(result.get("numBuckets").unwrap().is_u64());
-    assert!(result.get("numNodes").unwrap().is_u64());
-    assert!(result.get("numConnected").unwrap().is_u64());
+    let result = HistoryNetworkApiClient::routing_table_info(target).await;
+    assert!(result.is_ok());
```

Comment:
Here maybe we also want to check the `local_node_id` and `buckets` fields,

Reply:
I don't think an empty table is illegal if a node doesn't connect to a bootnode, the table should be empty.

Reply:
Alright. I'll go ahead and resolve this then